.ci/targets/android_build_all_packages.yaml

```diff
@@ -1,3 +1,5 @@
+# This builds legacy and modern all_packages app, which requires jdk11
+# This will stop working in the next stable!
```

Comment:
It'll stop working as-is, but will get updated. I'll adjust this comment.

Reply:
Actually, we can leave this as-is; I'll add a comment to the new target instead.